pkg/metadata/metadata.go

```diff
@@ -82,3 +88,43 @@ func ApplyAnnotations(c client.Client, ctx context.Context, node core.Node, anno
 	}
 	return nil
 }
+
+func ApplyDesiredVersionAnnotation(ctx context.Context, c client.Client, node core.Node, value string) error {
```

Comment:
does this need the value arg when it can just use version.Get() internally?
Or is there a dependency issue there?

Reply:
No dependency issue, updated

Reply:
`ApplyDesiredVersionAnnotation()` can rely on `version.Get()`, but note that we do have to pass in the value for `ApplyVersionAnnotation()` since it is used from WICD. Calling functions that do `version.Get()` from WICD results in an empty version value.